src/main/java/godaddy/registry/jtoolkit2/session/SessionManagerPropertiesImpl.java

```diff
@@ -215,6 +215,16 @@ public void setClientID(String id) {
         properties.setProperty("epp.client.clID", id);
     }
 
+    @Override
+    public byte[] getKeyStoreFileBytes() {
+        return null;
+    }
+
+    @Override
+    public byte[] getTrustStoreFileBytes() {
+        return null;
+    }
+
```

Comment:
It does not look like we need these two methods, please refer to our comment in `SessionProperties.java`.

Reply:
Same reason

Reply:
Removed two methods